fuzzing/common.bzl

```diff
@@ -99,3 +99,50 @@ specified in the srcs attribute.
         ),
     },
 )
+
+def _fuzzing_dictionary_impl(ctx):
+    output_dict = ctx.actions.declare_file(ctx.attr.output)
+    output_arg = "--output_file=" + output_dict.path
+    dict_args = "--dict_list=" + ctx.files.dicts[0].path
+
+    for single_dict in ctx.files.dicts[1:]:
+        dict_args += "," + single_dict.path
```

Comment:
Have you considered using https://docs.bazel.build/versions/master/skylark/lib/Args.html#add_joined?